docs/user/installation.rst

```diff
@@ -121,6 +128,28 @@ PyPi
 
 `intelmqsetup` will create all necessary directories, provides a default configuration for new setups. See the :ref:`configuration` for more information on them and how to influence them.
 
+Docker
+^^^^^^
+
+Navigate to your preferred installation directory, i. e. use ``mkdir ~/intelmq && cd ~/intelmq``
+
+.. code-block:: bash
+
+   git clone https://github.com/certat/intelmq-docker.git
+
+   sudo docker pull certat/intelmq-full:1.0
+
+   mkdir intelmq_logs
+
```

Comment:
If I understand https://github.com/waldbauer-certat/intelmq-docker/blob/main/docker-compose.yml correctly (forgive my ignorance) the folder `intelmq_logs` should be at the same level as `example_config`?

Reply:
change "should" to "could". You can mount whereever you want to. But I need to mount it somewhere in order to read logs.